.travis.yml

```diff
@@ -0,0 +1,38 @@
+# Based on https://github.com/NOAA-ORR-ERD/gridded/blob/master/.travis.yml
+language: minimal
+
+sudo: false
+
+notifications:
+  email: false
+
+matrix:
+  fast_finish: true
+  include:
+    - name: "docs"
+    - name: "compass"
+    #- name: "atmosphere"
+    #- name: "landice"
+    #- name: "ocean"
+    #- name: "seaice"
```

Comment:
To add more CI in the future, uncomment one of these or add a new one, then add `travis_ci/install_ocean.bash`, `travis_ci/test_ocean.bash`, etc. to set up and run whatever tests are desired.  It may be possible to set, compile and run MPAS itself in Travis is we don't make the tests too big.